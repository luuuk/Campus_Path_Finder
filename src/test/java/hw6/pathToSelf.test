# This test finds a path from a node to itself

LoadGraph test test.tsv

FindPath test a a