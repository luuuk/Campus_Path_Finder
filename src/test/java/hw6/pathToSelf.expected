# This test finds a path from a node to itself

loaded graph test

path from a to a: